aws/sdk/integration-tests/s3/tests/timeouts.rs

```diff
@@ -66,7 +67,7 @@ async fn test_timeout_service_ends_request_that_never_completes() {
         .await
         .unwrap_err();
 
-    assert_eq!(format!("{:?}", err), "TimeoutError(RequestTimeoutError { kind: \"operation timeout (all attempts including retries)\", duration: 500ms })");
+    assert_eq!("TimeoutError(TimeoutError { source: RequestTimeoutError { kind: \"operation timeout (all attempts including retries)\", duration: 500ms } })", format!("{:?}", err));
```

Comment:
I expect assertions to be in the format `assert_eq!(expected, actual)` so thanks for doing this!